### DISCUSSION TITLE
posts/back-forth/

### DISCUSSION THREAD

**Discussion by giscus** [brant-ruan/fernweh-comments#27] [Announcements]
# posts/back-forth/

我回家那年

https://blog.wohin.me/posts/back-forth/

**Comment by brant-ruan** [comment]
这是一篇从故纸堆里翻出来的即兴之作。那时妙手偶得一个玩弄文字的句子：我回家那年，他出家。于是顺着这句话往下延伸出了这些。没什么内容，也许多少能够体现出的，是刚刚启程探索宏大世界的迷茫与心志吧。
